model_compression_toolkit/trainable_infrastructure/pytorch/activation/lsq/symmetric_lsq.py

```diff
@@ -0,0 +1,110 @@
+# Copyright 2024 Sony Semiconductor Israel, Inc. All rights reserved.
```

Comment:
In principle there is, it even works, but it doesn't seem to survive squash merge. Regular merge will probably be ok. And it will require a new PR as the git tree will be completely different.
Alternatively I can remove the gptq fixes from the PR and only leave the minimal changes for the code relocation, and add a comment where it came from so that the old file's history can be viewed. At least we'll have a baseline for the future changes.

Reply:
Nevermind let's keep it like that, just let's talk offline where are the exact changes so I can review them